test/unit/OSTest.cc

```diff
@@ -38,7 +38,7 @@ TEST(OSTest, CreateSimOS) {
   EXPECT_GT(proc->context_.sp, 0);
```

Comment:
Should a test case be added for the case when a core is put to idle by process syscall result?